ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/422__caikit_py_client.py

```diff
@@ -0,0 +1,67 @@
+from caikit_nlp_client import HttpClient, GrpcClient
```

Comment:
temp file: to be used if you need/want to run tests against caikit-nlp-client library